apps/wallet-mobile/src/kernel/i18n/locales/en-US.json

```diff
@@ -212,8 +212,8 @@
   "components.pooltransition.poolGeneratesRewards": "This pool continues to generate staking rewards",
   "components.pooltransition.poolNoRewards": "This pool is NOT generating staking rewards anymore",
   "components.pooltransition.poolWillStopRewards": "This pool will stop generating rewards in",
-  "components.pooltransition.skipNoRewards": "Skip and stop receiving rewards",
-  "components.pooltransition.updateKeepEarning": "Update now and keep earning",
+  "components.pooltransition.skipNoRewards": "SKIP AND STOP RECEIVING REWARDS",
```

Comment:
@rahulnr7, that doesn't guarantee that other translations will follow the capitalization, it should be more effective to leverage `.toLocaleUpperCase()` on UI.

Reply:
Done. Thanks.